### DISCUSSION TITLE
Askit module issue

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#46] [1. Help]
Hi,

We are using Askit product question module and there seems an issue when we log in from the product page when we try to answer as a guest, the issue on the landing page where we land after login, there is being appended full page again on that page

Please see the below screen where you can see 404 page coming inside page.

![Screenshot from 2023-01-11 14-24-49](https://user-images.githubusercontent.com/19511407/211761475-ff33dbcf-d1f8-4110-a6c0-3a97321d545e.png)

We are using this with a breeze theme.

I checked further and found the issue only with the Breeze theme, it's working fine with luma.

Thanks,
Pankaj

**Comment by vovayatsyuk** [comment]
Made a fix in ajax response processing: https://github.com/breezefront/module-breeze/commit/6e485663d01a4323fef69bf11014708a1ef86faf

Now the ajax requests will be failed if the response is not 2xx (this fixes nested pages).

However, additional bugs should be fixed in the Askit module (incorrect redirect URL). I've made a ticket in the Askit repository. You can also contact swissuplabs support about incorrect redirect after login via askit links.

**Reply by pk-pankaj** [reply]
Thanks @vovayatsyuk
